### DISCUSSION TITLE
Barcode / timeline sometimes appears ±2x too long

### DISCUSSION THREAD

**Discussion by avdiem** [videovillage/community#117] [Ideas]
When playing a movie from a URL (e.g. YouTube), I sometimes get a barcode / timeline that appears approx. 2x longer than the actual playing time. When the video reaches the end, the playhead has progressed to only ±50% of the barcode / timeline: 

<img width="1498" alt="Screenshot 2020-08-15 at 11 33 17" src="https://user-images.githubusercontent.com/34418922/90309788-c4e14580-deeb-11ea-9344-2fb2fbbfcabc.png">

I’ve just checked videos that I’ve captured before; only some of these seem to have this, not all.

**Comment by gregcotten** [comment]
Hi there! Yes, this is an issue with certain videos, and it has something to do with the pulled-down content reporting incorrect time ranges. Looking into it now!
